doc/README.md

```diff
@@ -471,6 +471,34 @@ currently selected entry is marked with a ✓. Clicking on the 🗙 on the
 right-hand side of such an entry deletes the configuration (after
 prompting the user to confirm).
 
+## Flame graph
+
+The `Flame graph (new)` view displays profile information as a
+[flame graph](https://www.brendangregg.com/flamegraphs.html).
+
+Boxes on this view corresponding to stack frames in the profile. Caller boxes
+are directly above callee boxes. The width of each box is proportional to the
+number of profile samples where that frame was present on the call stack.
+Children of a particular box are laid out left to right in decreasing size
+order.
+
+Inlining is indicated by the absence of a horizontal border between a caller and
```

Comment:
optional: I would move this paragraph down to be last as inlining is a less common concept people would want to learn than naming of the boxes and coloring.